scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -910,7 +911,7 @@ String getTaskServerLogsByTag(String id, String taskTag) throws UnknownJobExcept
      * @throws NotConnectedException if you are not authenticated.
      * @throws PermissionException if you have not enough permission to access this method.
      */
-    List<JobInfo> getJobs(int offset, int limit, JobFilterCriteria filterCriteria,
+    Page<JobInfo> getJobs(int offset, int limit, JobFilterCriteria filterCriteria,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.PermissionException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)